pom.xml

```diff
@@ -5,6 +5,19 @@
   <artifactId>cruler</artifactId>
```

Comment:
I suggest changing `<groupId>` to `xcoo` because both [the official doc](https://clojure.org/guides/deps_and_cli) and [clojure style guide](https://guide.clojure.style/#lib-coordinates) recommend that. And besides, recent xcoo's libraries use `xcoo` as `<groupId>`, e.g. [`xcoo/clj-build-date`](https://clojars.org/xcoo/clj-build-date) and [`xcoo/integrant-tools`](https://clojars.org/xcoo/integrant-tools).

Reply:
I see! fix https://github.com/xcoo/cruler/pull/7/commits/32c0bcd5a91274d3ea8f899099f8abb602d0b47e